packages/fluentui/react-northstar/test/specs/components/Form/FormButton-test.tsx

```diff
@@ -8,5 +8,6 @@ describe('FormButton', () => {
     constructorName: 'FormButton',
     forwardsRefTo: `Button`,
     passesUnhandledPropsTo: Button,
+    disabledTests: ['component-handles-classname'],
```

Comment:
Can you please clarify why this is disabled? The same test worked before and now is broken? That looks suspicious 🤔

Reply:
I investigated the disabled Form tests and they are now all enabled. Here is what happened:

Reason Why It Failed:
•	The `passessUnhandledPropsTo` type was acting incorrectly in `component-handles-classname` and was behaving as an incorrect workaround for components that didn’t apply classNames to the root.

Solution:
•	I decided to remove usage of `passessUnhandledPropsTo` in the classNames test since it was only added to OverflowSet.

•	After removing the usage of `passessUnhandledPropsTo` I fixed the bug in `OverflowSet` (placement of `getNativeProps`) and enabled the test.